Client/Frontend/Browser/URLBarView.swift

```diff
@@ -276,6 +276,16 @@ class URLBarView: UIView, BrowserLocationViewDelegate, AutocompleteTextFieldDele
         newTabsButton.titleLabel?.layer.cornerRadius = 2
         newTabsButton.titleLabel?.font = AppConstants.DefaultSmallFontBold
         newTabsButton.titleLabel?.textAlignment = NSTextAlignment.Center
+
+        self.addSubview(newTabsButton)
+
+        // copy constraints from original button for positioning
+        newTabsButton.snp_makeConstraints { make in
+            make.centerY.equalTo(self.locationContainer).offset((URLBarViewUX.TabsButtonRotationOffset - 0.5) * URLBarViewUX.TabsButtonHeight)
+            make.trailing.equalTo(self)
+            make.width.height.equalTo(AppConstants.ToolbarHeight)
+        }
```

Comment:
I think you should be able to use the actual constraints from `tabsButton` instead of duplicating them here:
```
newTabsButton.snp_makeConstraints { make in
    make.edges.equalTo(self.tabsButton)
}
```